airbyte-integrations/singer/base-singer/base_singer/singer_helpers.py

```diff
@@ -31,7 +33,7 @@ def discover(shell_command, transform=(lambda x: AirbyteSchema(x))) -> AirbyteSc
         return transform(completed_process.stdout)
 
     @staticmethod
-    def read(shell_command, is_message=(lambda x: True), transform=(lambda x: AirbyteMessage(x))) -> Generator[
+    def read(shell_command, is_message=(lambda x: True), transform=(lambda x: AirbyteMessage(type="RECORD", record=AirbyteRecordMessage(data={'value': x}, emitted_at=str(datetime.now()))))) -> Generator[
```

Comment:
This will probably change to just outputting an object that will convert to json, which will be placed into the data field. Then `base.py` will be responsible for setting the timestamp and constructing the objects.